csharp/OnnxRuntime.CSharp.proj

```diff
@@ -144,6 +147,87 @@ CMake creates a target to this project
     />
   </Target>
 
+  <!-- To support build.py and CI easily, generate the sub packages in the proj -->
+  <Target Name="CreateSubGpuWinPackages" AfterTargets="CreatePackage" Condition="'$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime.Gpu'">
+    <!-- Manually create the nuspec for the native Microsoft.ML.OnnxRuntime package -->
+    <Message Importance="High" Text="Generating nuspec for the native Microsoft.ML.OnnxRuntime nuget package..." />
+    <Exec Command="$(PythonExe) $(GenerateNuspecScript) --package_version $(PackageVersion) --package_name Microsoft.ML.OnnxRuntime.Sub.Gpu.Windows --target_architecture $(TargetArchitecture) --build_config $(Configuration) --native_build_path $(NativeBuildOutputDirAbs) --packages_path $(OnnxRuntimePackagesDirectoryAbs) --ort_build_path $(OnnxRuntimeBuildDirectoryAbs) --sources_path $(OnnxRuntimeSourceDirectoryAbs) --commit_id $(GitCommitHash) --is_release_build $(IsReleaseBuild) --execution_provider $(ExecutionProvider)"
```

Comment:
Comment/message needs updating here and line 192 - 194.

I find the package name a little confusing due to the ordering. Having 'OnnxRuntime.Sub.Gpu' makes me think it's related to the base OnnxRuntime package. Microsoft.ML.OnnxRuntime.Gpu.Sub.Windows would have a clearer association with Microsoft.ML.OnnxRuntime.Gpu.

Having thought about this a little more and aking a step back, would it be simpler if we instead made both the platform specific packages standalone (include headers, props and targets files in both) and call them Microsoft.ML.OnnxRuntime.Gpu.Windows and Microsoft.ML.OnnxRuntime.Gpu.Linux? We would still have Microsoft.ML.OnnxRuntime.Gpu with a dependency on both for backwards compatibility and ease of use, but it would only provide those dependencies. That setup avoids having to try and explain what a 'sub' package is to confused users.

Reply:
Do you mean that we should support Microsoft.ML.Onnxruntime.Gpu.Windows be installed alone?

Reply:
Now, the Onnxruntime.GPU.Windows/Linux can work indivisually.